linux_os/guide/system/logging/journald/socket_systemd-journal-remote_disabled/bash/shared.sh

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_ubuntu
+
+{{{ bash_package_install("systemd-journal-remote") }}}
```

Comment:
I think this line should be removed. Installation of the package is definitely out of scope for this rule.

Reply:
removed it